packages/global/src/config/icons/database.ts

```diff
@@ -259,7 +259,6 @@ export const DatabaseSearchClose = html`<svg
   width="16"
   height="16"
   viewBox="0 0 16 16"
-  fill="none"
```

Comment:
We can keep `fill="none"` here.

Reply:
It seems when keep fill="none" here, and add  fill: var(--affine-black-90) in .close-icon, the icon will be still not visible. Any suggestion to fix it?

Reply:
That's fine too. Alternatively, you can set the fill style like `.close-icon > svg`.

Reply:
OK. Thanks for your advice.